Find a functionally equivalent code:android.view.HardwareRenderer.trimMemory:COMMENT
Method Modifier: default     static      
Comment:/**
 * Invoke this method when the system is running out of memory. This
 * method will attempt to recover as much memory as possible, based on
 * the specified hint.
 *
 * @param level Hint about the amount of memory that should be trimmed,
 * see {@link android.content.ComponentCallbacks}
 */

@@ -1,4 +1,3 @@
 {
-    startTrimMemory(level);
-    endTrimMemory();
+    ThreadedRenderer.trimMemory(level);
 }

